package.json

```diff
@@ -27,7 +27,8 @@
     "prepublish": "cross-env NODE_ENV=production npm run build",
     "lint": "eslint webpack.config.js src/**/*.js",
     "format": "prettier --write \"**/*.js\"",
-    "test": "cross-env BABEL_ENV=commonjs jest --no-cache"
+    "test": "cross-env BABEL_ENV=commonjs jest --no-cache",
+    "react": "enzyme-adapter-react-install 16.2.0"
```

Comment:
there is a bug in semver.intersect that enzyme-adapter-react-helper uses. Once that is resolved, we will be able to use a wider semver as opposed to locking it down a specific version